doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -289,6 +290,25 @@ Apart from the `status` values defined for subscription in the NGSIv2 specificat
 
 [Top](#top)
 
+## `forcedUpdate` option
+As extra URI param option to the ones included in the NGSIv2 specification, Orion implements forcedUpdate, 
+than can be used to specify that an update operation have to trigger any matching subscription (and send 
+corresponding notification) no matter if there is an actual attribute update or not. Remember that the 
+default behaviour (i.e. without using the forcedUpdate URI param option) is to updated only if attribute 
+is effectively updated.
+
+The following requests can use the forcedUpdate URI param option:
+* `POST /v2/entities?options=forcedUpdate`
```

Comment:
> Although it is not included in the list of the issue, I have just realized that POST /v2/entities?options=forcedUpdate should be also included. Please, add it to this list in the documenation and to the .test (if not already included).

I have realized that I have said something without sense :$

forcedUpdate doesn't make sense in create entity operation, given that there isn't any previous entity to which compare the value. Please, remove that like from documentation and don't take it into acocunt in .test

Sorry the noise... :(

Reply:
Fixed in 7a776db